app/models/package.js

```diff
@@ -42,12 +42,12 @@ export default Model.extend(cloudinaryImage, {
   isDispatched: Ember.computed.bool("stockitSentOn"),
 
   isAvailable: Ember.computed("isDispatched", "allowWebPublish", function() {
-    return (
+    return Boolean(
```

Comment:
Before it wasn't returning boolean ?

Reply:
Same question @patrixr , What was it returning before(if not boolean)?

Reply:
Javascript has the concept of things being "truthy", which allows us to do things like this :

```javascript
const str = "a string";
if (str) {
   ...
}
```

But the result of the expression is still not a boolean.
In the case of an `&&` expression, the value will equal the first "falsy" result. If all of them are "truthy" then the last value is used

Meaning : 

```
1 && 2 && 0 && 4  => 0
1 && 2 && "hello" => "hello"
"hello" && 2 && 1 => 1
```

Here the value would be `this.get("quantity")` which is an integer.
Which would fail later when using code suche as `.filterBy('isAvailable', true)` (because 1 !== true)

Reply:
❤️ All hail Typescript ❤️

Reply:
great explanation @patrixr thanks :) 👍

Reply:
😄